src/engine/instrument.cpp

```diff
@@ -2172,6 +2205,17 @@ void DivInstrument::readFeatureS2(SafeReader& reader, short version) {
   READ_FEAT_END;
 }
 
+void DivInstrument::readFeatureO4(SafeReader& reader, short version) {
```

Comment:
Change. This should be part of MultiPCM ins struct.

In the future we might have DX7 and require O4-O5.
Also, very likely conflicts with operator 4 macros.